### DISCUSSION TITLE
[Feature Request] Add support for Paper's AsyncChat

### DISCUSSION THREAD

**Discussion by NoltoxGit** [QuickShop-Community/QuickShop-Hikari#1344] [Ideas]
Hello,
Can you add the support for the Paper `AsyncChatEvent` and/or `AsyncChatDecorateEvent` in the configuration of your plugin please?
Because you're currently using an old listener: `org.bukkit.event.player.AsyncPlayerChatEvent`. 

Example for a new configuration:
```yml
# Use Paper's async Chat Event
use-async-chat: true/false
```

Thank you so much if you add this! :)

**Comment by Ghost-chu** [comment]
Can you give a reason why this feature is needed?QuickShop does not modify chat messages sent by players, it is only used for input.
